python/py-sqlparse/Portfile

```diff
@@ -13,16 +13,16 @@ description         Non-validating SQL parser
 long_description    ${description} that provides support for parsing, splitting \
                     and formatting SQL statements.
 
-python.versions     27 34 35 36
+python.versions     27 34 35 36 37
 
 categories-append   textproc
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  6264e56cbfdca0273b11c0f6a3334891ac9c574b \
-                    sha256  d446296b2c26f9466860dd85fa32480bec523ab96bda8879262c38e8e8fbba21
+checksums           rmd160  cbc1000e0fea6334d1981e6b5bee1d9dd95b8102 \
+                    sha256  ce028444cfab83be538752a2ffdb56bc417b7784ff35bb9a3062413717807dec
```

Comment:
It is now customary to put in a "size" field in addition to the rmd160 and sha256.